credentials/credentials.go

```diff
@@ -287,5 +287,5 @@ type ChannelzSecurityValue interface {
 type OtherChannelzSecurityValue struct {
 	ChannelzSecurityValue
 	Name  string
-	Value proto.Message
+	Value protoadapt.MessageV1
```

Comment:
this could just be 
```suggestion
	Value proto.message
```
of `google.golang.org/protobuf/proto` if you also fix the usage in https://github.com/grpc/grpc-go/blob/d41b01db97ca2e3627b2c9949fffe8f152a4255d/channelz/service/service.go#L193 to use Value as is